docs/dasharo-tools-suite/documentation.md

```diff
@@ -692,14 +692,10 @@ Serving HTTP on 0.0.0.0 port 9000 (http://0.0.0.0:9000/) ...
 
 #### Run DTS using VentoyOS
 
-Starting from version v1.1.1, we provide also an ISO formatted image. Thanks to
-that, it can be used with VentoyOS[1]. As for now the following limitations are
-known.
-
-* VentoyOS needs to be started in UEFI mode.
-* Nothing can be saved on root file system, as VentoyOS boots systems in
-  read-only mode.
-
-Please let us know if you started DTS using VentoyOS and have additional
-information for us. You can share them on [Dasharo Matrix
-Workspace](https://matrix.to/#/#dasharo:matrix.org).
+The functionality of DTS is closely linked to the capability of writing files to
+the root file system. However, utilizing VentoyOS for booting enables read-only
```

Comment:
I think it's the problem of ISO itself, not ventoy. I have been using just ISo o ver PiKVM with the same problem. Usere have been reporting writing ISOs on USB with the same problem. So ventoy is not the one to blame here.

Reply:
@macpijan valid point. Maybe this has more sense now https://github.com/Dasharo/docs/pull/791/commits/e8a357e0ae2201af1fd98edd203b7d36e1fa2aab

Reply:
LGTM